packages/venia-ui/lib/components/Navigation/navigation.js

```diff
@@ -76,9 +76,11 @@ const Navigation = props => {
         setView('SIGN_IN');
     }, [setView]);
 
+    const isRootClass = isTopLevel && view === 'MENU' ? classes.isRoot : null;
```

Comment:
@revanth0212 : I have modified the name as "menuTopLevelClassName" instead of "rootClassName" as it was already given to "aside".
Have also removed the null and replaced with ' '.

Please review.